fabric/v4/model_cloud_router_action_type.go

```diff
@@ -3,18 +3,17 @@
  *
  * Equinix Fabric is an advanced software-defined interconnection solution that enables you to directly, securely and dynamically connect to distributed infrastructure and digital ecosystems on platform Equinix via a single port, Customers can use Fabric to connect to: </br> 1. Cloud Service Providers - Clouds, network and other service providers.  </br> 2. Enterprises - Other Equinix customers, vendors and partners.  </br> 3. Myself - Another customer instance deployed at Equinix. </br>
  *
- * API version: 4.9
+ * API version: 4.10
  * Contact: api-support@equinix.com
  * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
  */
 package v4
 
-// GatewayActionState : Cloud Router action state
-type GatewayActionState string
+// CloudRouterActionType : Cloud Router action type
+type CloudRouterActionType string
 
-// List of GatewayActionState
+// List of CloudRouterActionType
 const (
-	DONE_GatewayActionState    GatewayActionState = "DONE"
-	FAILED_GatewayActionState  GatewayActionState = "FAILED"
-	PENDING_GatewayActionState GatewayActionState = "PENDING"
+	BGP_SESSION_STATUS_UPDATE_CloudRouterActionType CloudRouterActionType = "BGP_SESSION_STATUS_UPDATE"
```

Comment:
I don't know why the diff rendered this way, but for some reason `git` decided that GatewayActionState became CloudRouterActionType instead of CloudRouterActionState.  That makes it look like there are major enum changes in this release.

Reply:
The diff should have compared GatewayActionState to CloudRouterActionState; the old GatewayActionState code is above this comment, and the new CloudRouterActionState code is here: https://github.com/equinix-labs/fabric-go/pull/17/files#diff-564860f61a10f2726340a09cd95a2e0bb507c481e5ad00acf65d1b835f5a3fb6R17-R19

Reply:
That's something I've noticed about the SDK updates as well. It's a little funky but it does end up in our desired state. We've updated the changes in the description of the PR and are moving forward with it. Thanks for the comments, Charles!